sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -374,6 +374,24 @@ object RapidsConf {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(15L * 1024 * 1024) // 15 MiB
 
+  val EXECUTOR_OVERHEAD_SIZE = conf("spark.executor.memoryOverhead")
```

Comment:
I saw that we initialize our `RapidsConf` with the `sparkConf` before passing it to the `GpuDeviceManager`, but I'm not sure the best way to access those confs, I'm guessing this is not it.

Reply:
The configs here are a little odd. For example, if we have a default here, but it is different than spark's default, what happens? Does spark get one value and we get another?

I would use the config directly from the SparkConf object. `SparkEnv.get.conf` instead.